infra/modules/kubevela-addons/definitions/cardano-wallet-connector.cue

```diff
@@ -0,0 +1,171 @@
+parameter: {
+	network:       *"preview" | "preprod" | "mainnet"
+}
+
+#cardanoWalletConfigs: {
+  volumes: [{
+    name: "wallet-db"
+    persistentVolumeClaim: {
+      claimName: "wallet-db"
+    }
+  }, {
+    name: "db"
```

Comment:
This has nothing to do with `cardano-wallet`, this is the state dir for the dapps-certification service.